### ISSUE TITLE
Plotter: misc. detail improvements

### ISSUE THREAD

**Issue by AlexTate** [MontgomeryLab/tinyRNA#161]
- [x] ~Change the way scatter axis labels are handled so that they behave as a contiguous PDF text elements. Currently, due to the inline mathtext for the Log<sub>2</sub> subscript, the label strings are tokenized into individual PDF text elements per-character. This makes it challenging for users to edit these axes labels in PDF editors as one might wish to do for publications.~
- [x] Change the font utilized by the Log<sub>2</sub> mathtext subscript. MPL is switching fonts for this particular character, and the font used is not common. This leads to missing font errors when editing plot PDFs.
- [x] ~Make scatter points darker for replicate scatter plots and non-class DGE scatter plots~

**Comment by AlexTate** [comment]
Matplotlib supports a subset of LaTeX that they call "mathtext", and this subset is baked into the package (does not require any additional LaTeX dependencies). I've corrected the above usages of "TeX" to "mathtext"

Unfortunately an ideal solution couldn't be found for the axis label tokenization issue. I've tried the following:
- Change the mathtext subscript 2 to a unicode subscript 2, and switch the font to a set that supports unicode characters (in our case this was Arial -> Arial Unicode MS, which was conveniently provided by our mscorefonts conda dependency). Unfortunately this increased PDF file sizes by 15x and runtimes by 3x. Unicode font sets cover a substantially wider character range and are therefore larger resource files, which need to be embedded in the PDF.
- Using the previous method but changing the PDF font type from TrueType to Type 3. This led to very lightweight PDFs (smaller than those currently produced by the master branch) and eliminated the 3x runtime increase, but it led to the tokenization of every text field (not just axis labels).
- Make the entire axis label mathtext rather than interleaving mathtext with regular text. Axis label tokenization was still present and spaces were not rendered.
- The PGF backend will supposedly subset fonts. However, this requires an external LaTeX dependency and the project footprint is already fairly large. It didn't seem like an appropriate tradeoff
